modules/utxo-lib/test/bitgo/fixtures/signature/bitcoin/p2tr/parsed-backup-bitgo-fullSigned.json

```diff
@@ -7,8 +7,8 @@
     "e21c29b4a7eeace9c7a8cefb568ca00c86ff9bf5e79e07e5442c29d4a0950d04"
   ],
   "signatures": [
-    "9f74390b6c7f5988772a909fc689a3e2dd680ceb8d54d4caffdd5907132bc415d7d964f4b25ce02690938cd4b62057f80b929aca4155c64db205a336b2e6ce79",
-    "69a969c8df93bc7894d045bc5e5d73f50d815fed7f703f11080f6b46561220156a0eed3382ec0eddbfa2737a29a7b07ba572880ae26f05f56ebca40bd7a5a987"
+    "a2074b68b115e970c8c4bee9cf75c03b82bd3e3086d58801a4adb7b31b9b5f2cea4a5615a4feef74b61db6d765bb814feaf27331fad128dda55cfe106d25d44e",
```

Comment:
Are these changes all because of the lack of extra nonce data?

Reply:
Yes